Include/AndroidExtensions/JavaWrappers.h

```diff
@@ -166,6 +173,36 @@ namespace java::lang
     };
 }
 
+namespace java::websocket
+{
+    class WebSocketClient : public lang::Object
+    {
+    public:
+        WebSocketClient(std::string url, std::function<void()> open_callback, std::function<void()> close_callback, std::function<void(std::string)> message_callback, std::function<void()> error_callback);
+        ~WebSocketClient();
+        void Send(std::string message);
+        void Close();
+
+        static jclass g_webSocketClass;
```

Comment:
Does this need to be public? Also naming

```suggestion
        static jclass s_webSocketClass;
```